ui/app/AppLayouts/Wallet/panels/ManageCollectiblesPanel.qml

```diff
@@ -58,6 +58,12 @@ Control {
             sourceModel: d.renamedModel
             arrangeByCommunity: switchArrangeByCommunity.checked
             settingsKey: "WalletCollectibles"
+            onTokenHidden: (symbol, name) => Global.displayToastMessage(
```

Comment:
Not sure the `symbol` here is storing the information we really want to show. See video for better understanding:

https://github.com/status-im/status-desktop/assets/97019400/41aff7bb-7c57-43b1-aad0-1d3dbb914ed6

Reply:
Looking at `CollectiblesView` it seems the `subtitle` property contains: `subTitle: model.collectionName ? model.collectionName : model.collectionUid ? model.collectionUid : ""`

Reply:
Ah right, in case of a collectible we probably don't want/need the symbol... will fix

Reply:
Fixed